pkg/engine/files.go

```diff
@@ -46,11 +47,11 @@ func newFiles(from []*chart.File) files {
 //
 // This is intended to be accessed from within a template, so a missed key returns
 // an empty []byte.
-func (f files) GetBytes(name string) []byte {
+func (f files) GetBytes(name string) ([]byte, error) {
```

Comment:
`GetBytes` is a public function in the SDK. It's signature cannot be changed until we get to Helm v4. We only do backwards compatible feature additions. The same applied for all the functions in Helm.

Can you please revert any sig changes.

Reply:
I applied this change because the current behaviour is that if you try to access a file that isn't included - the engine will treat it as empty. This behaviour is a bit unexpected and can make it hard to understand why the chart isn't rendered as expected.
It's more important now when this behaviour can happen because the user didn't include the file he wanted to include.

Is there perhaps a better way we can deal with this without breaking the API?

Reply:
Maybe it would be enough to just log a warning message for now?
Do you think though that we can apply this change in Helm 4.0?

Reply:
You can introduce a new function. But you cannot change existing function signatures.